sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/api/Azure.ResourceManager.DeviceUpdate.netstandard2.0.cs

```diff
@@ -271,8 +271,8 @@ namespace Azure.ResourceManager.DeviceUpdate.Models
     public partial class CheckNameAvailabilityRequest
     {
         public CheckNameAvailabilityRequest() { }
+        public string CheckNameAvailabilityRequestType { get { throw null; } set { } }
```

Comment:
This should be named ResourceType.

Reply:
The generator is updated to automatically rename Type to ResourceType in CheckNameAvailabilityRequest.